services/121-service/src/registration-events/registration-events.controller.ts

```diff
@@ -66,59 +64,55 @@ export class RegistrationEventsController {
     description:
       'Format to return the data in. Options are "json" and "xlsx". Defaults to "json" if not specified. If "xlsx" is selected, the response will be a file download in which the data is slightly differently formatted for portal users.',
   })
-  @Get('programs/:programId/registration-events')
+  @Get('programs/:programId/registration-events/export')
   public async getEvents(
     @Param('programId', ParseIntPipe) programId: number,
     @Query() queryParams: GetRegistrationEventsQueryDto,
     @Query('format') format = 'json',
-    @Req() req: ScopedUserRequest,
     @Res() res: Response,
   ): Promise<GetRegistrationEventDto[] | void> {
-    // TODO: REFACTOR: nothing actually happens with this filename, it is overwritten in the front-end
-    const filename = `registration-data-change-events`;
     const searchOptions = {
       queryParams,
     };
-    const errorNoData = 'There is currently no data to export';
-    if (format === ExportFileFormat.xlsx) {
-      const result = await this.registrationEventsService.getEventsAsXlsx({
+    const result =
+      await this.registrationEventsService.getRegistrationEventsExport({
         programId,
         searchOptions,
       });
-      if (result.length === 0) {
-        throw new HttpException({ errors: errorNoData }, HttpStatus.NOT_FOUND);
-      }
-      return sendXlsxReponse(result, filename, res);
+    if (result.data.length === 0) {
+      const errorNoData = 'There is currently no data to export';
+      throw new HttpException({ errors: errorNoData }, HttpStatus.NOT_FOUND);
     }
 
-    const result = await this.registrationEventsService.getEventsAsJson({
-      programId,
-      searchOptions,
-    });
-    if (result.length === 0) {
-      throw new HttpException({ errors: errorNoData }, HttpStatus.NOT_FOUND);
+    if (format === ExportFileFormat.xlsx) {
+      // TODO: REFACTOR: nothing actually happens with this filename, it is overwritten in the front-end
+      const filename = `registration-data-change-events`;
+      return sendXlsxReponse(result.data, filename, res);
     }
     res.send(result);
   }
 
   @AuthenticatedUser({ permissions: [PermissionEnum.RegistrationPersonalREAD] })
   @ApiOperation({
-    summary: '[SCOPED] Get paginated registration-events for this programId',
+    summary:
+      '[SCOPED] Get registration-events for the monitoring page (excludes type=status-change)',
   })
   @ApiParam({ name: 'programId', required: true, type: 'integer' })
   @PaginatedSwaggerDocs(
     RegistrationEventViewEntity,
     PaginateConfigRegistrationEventView,
   )
-  @Get('programs/:programId/registration-events/paginated') // ##TODO: change api-path and/or merge with existing endpoints
-  public async getRegistrationEventsPaginated(
+  @Get('programs/:programId/registration-events/monitoring')
```

Comment:
Hmm maybe we should do:

```suggestion
  @Get('programs/:programId/registration-events')
```

 In the function names I find it quite clear that one path is named `monitoring` and the other one is named `export`. However I wonder if in this case it is not better to stick to default api path naming. 

Let's discuss